src/database/accounts.js

```diff
@@ -51,29 +51,16 @@ module.exports = {
     },
 
     /**
-     * Search for a user by name
+     * Search for a user by any field
      */
-    search: function (name, fields, callback) {
-        /* This bit allows it to accept varargs
```

Comment:
I couldn't find this actually been used anywhere. And wouldn't it be better to just package the vars as an object for such a use case?